lib/omise/res/OmiseApiResource.php

```diff
@@ -250,7 +250,7 @@ private function _executeTest($url, $requestMethod, $key, $params = null)
      */
     private function genOptions($requestMethod, $userpwd, $params)
     {
-        $user_agent        = "OmisePHP/".OMISE_PHP_LIB_VERSION;
+        $user_agent        = "OmisePHP/" . OMISE_PHP_LIB_VERSION . " PHP/" . phpversion();
```

Comment:
Note that the version string returned by phpversion() may include more information than expected: "5.5.9-1ubuntu4.17", for example. [source](http://php.net/manual/en/function.phpversion.php#119607)